infra.bs

```diff
@@ -630,6 +630,13 @@ out-of-bounds, except when used with <a for=list>exists</a>.
 "<code>b</code>", "<code>c</code>", "<code>a</code>" ». Then |example|[1] is the <a>string</a>
 "<code>b</code>".
 
+<p>Additionally, a literal syntax can be used to express a <a>list</a> of integers: the range
+<var>n</var> to <var>m</var>, inclusive, creates a new <a>list</a> whose <a for=list>items</a> are
+all integers in the range <var>n</var> to <var>m</var>, inclusive.
```

Comment:
One alternative is to do what is common in math textbooks and say something like "the list « _n_, _n_ + 1, ..., _m_ - 1, m »". But that notation (the ... part in particular) is undefined, and has some magic where in fact the list could contain as few as one items despite looking like it contains at most four.

Maybe...

> creates a new ordered set containing all of the integers from _n_ up to and including _m_

which is somewhat saying the same thing in different words, but works OK.

Also the definition should state that it's "for _m_ &geq; _n_".